### PR TITLE
Code to slurp in WMKO spectra of standard stars for DEIMOS

### PR DESCRIPTION
**Pull request by profxj**
Basically a simple set of code to handle the IDL output from
Greg Wirth's scripts.

The PR includes a test that runs `sensfunc`.

### CODE REVIEW

File: pypeit/specobj.py [hunk 1]
```diff
@@ -25,6 +25,7 @@
 from pypeit import utils
 from pypeit import datamodel
 from pypeit.images import detector_container
+from pypeit.spectrographs.util import load_spectrograph
```
Comment (jhennawi) [inline pypeit/specobj.py]: I think this can be removed since you don't require spectrograph.

File: pypeit/specobj.py [hunk 2]
```diff
@@ -117,7 +118,7 @@ class SpecObj(datamodel.DataContainer):
                                           descr='Boxcar extracted noise variance, sky+read noise '
                                                 'only (counts^2)'),
                  'BOX_MASK': dict(otype=np.ndarray, atype=np.bool_,
-                                  descr='Mask for optimally extracted flux'),
+                                  descr='Mask for optimally extracted flux. True=good'),
```
Comment (badpandabear) [inline pypeit/specobj.py]: Should this be "...boxcar extracted flux..."?

**Review by badpandabear — APPROVED** [review]

File: pypeit/specobj.py (hunk 2 above)
Comment (profxj) [reply pypeit/specobj.py]: good catch

File: pypeit/specobj.py (hunk 1 above)
Comment (profxj) [reply pypeit/specobj.py]: agreed; gone

**Review by jhennawi — APPROVED** [review]
Great work!

**Pull request merged**